x-pack/plugins/synthetics/e2e/page_objects/synthetics_app.tsx

```diff
@@ -63,9 +62,31 @@ export function syntheticsAppPageProvider({ page, kibanaUrl }: { page: Page; kib
       return await this.findByText('Monitor added successfully.');
     },
 
+    async deleteMonitors() {
+      // const isSuccessful: boolean = false;
+      while (true) {
+        if ((await page.$(this.byTestId('syntheticsMonitorListActions'))) === null) break;
+        await page.click(this.byTestId('syntheticsMonitorListActions'), { delay: 800 });
+        await page.click('text=delete', { delay: 800 });
+        await page.waitForSelector('[data-test-subj="confirmModalTitleText"]');
+        await this.clickByTestSubj('confirmModalConfirmButton');
+        // isSuccessful = Boolean(await this.findByTestSubj('uptimeDeleteMonitorSuccess'));
```

Comment:
I was having trouble with monitor deletion, but I think it's been resolved. Need to update this.